docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -57,3 +57,58 @@ Result:
 │       6 │
 └─────────┘
 ```
+
+## seriesDecomposeSTL
+
+Decompose time series data based on STL(Seasonal-Trend Decomposition Procedure Based on Loess)
```

Comment:
`Decomposes a time series using STL (Seasonal-Trend Decomposition Procedure Based on Loess)](https://www.wessa.net/download/stl.pdf) into a season, a trend and a residual component.'